docs/api/heat/boundary_conditions.rst

```diff
@@ -1,10 +1,10 @@
 .. currentmodule:: tidy3d
 
 Thermal Boundary Conditions
-===============================
+--------------------
```

Comment:
I thought these underlines must be longer than the title?

Reply:
Yeah all done now